IdentityCore/src/MSIDDeviceId.m

```diff
@@ -120,8 +154,35 @@ + (NSString *)deviceOSId
     return s_OSString;
 }
 
-+ (void)setIdValue:(NSString*)value
-            forKey:(NSString*)key
++ (NSString *)deviceTelemetryId
```

Comment:
Should we consider to name this method as "deviceId" and rename current "deviceId" to something like "deviceInfo" or "deviceDiagnosticInfo"?

Reply:
Makes sense. Will do it as part of https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/issues/17